### PR TITLE
Add DotMenu component

### PR DESCRIPTION
**Pull request by ash3rz**

### CODE REVIEW

File: stories/DotMenu.stories.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    >
+                        Item 3
+                    </MenuItem>,
+                ]}
+            />
+        );
+    }
+}
+
+storiesOf("DotMenu", module).add("with sample menu items", () => (
```
Comment (psarando) [inline stories/DotMenu.stories.js]: Does the `ui-lib` support the new style `Component Story Format` that are simple exports now?

File: src/components/dotMenu/DotMenu.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+/**
+ *  @author aramsey
+ *
+ *  A component which displays 3 vertical dots, which when
+ *  clicked will display a menu.
+ *
+ *  MenuItems should be passed into the render prop which in turn
+ *  will provide a function for closing the menu.
+ *
+ *  Sample usage:
```
Comment (psarando) [inline src/components/dotMenu/DotMenu.js:10]: Good comments 👍

File: src/components/dotMenu/DotMenu.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+import React, { useState } from "react";
+
+import { Menu, IconButton } from "@material-ui/core";
+import { MoreVert } from "@material-ui/icons";
+
+import build from "../../util/DebugIDUtil";
+
+function DotMenu(props) {
+    const { baseId, render, ButtonProps, MenuProps } = props;
```
Comment (psarando) [inline src/components/dotMenu/DotMenu.js:31]: 👍

**Review by psarando — APPROVED** [review]
LGTM :shipit:

File: stories/DotMenu.stories.js (hunk 1 above)
Comment (ash3rz) [reply stories/DotMenu.stories.js]: Ah, good catch.  I'll push a change to update to the new format.  It requires a slight change to the storybook config, but it doesn't break the stories in the old format.

**Comment by ash3rz** [comment]
Thanks for the review!

**Pull request merged**